### PR TITLE
Fixes documentation with respect to 3D mouse controls on Mac

### PR DESCRIPTION
**Pull request by omichel**
Fixes #2640.

See the result at https://cyberbotics.com/doc/guide/the-3d-window?version=fix-documentation-keyboard-shortcuts

I also took the opportunity of this PR to fix the appearance (CSS) of the `kbd` tag (keyboard keys).

### CODE REVIEW

File: docs/guide/the-scene-tree.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ Additionally, if the current selection is a [Robot](../reference/robot.md) node
 
 Nodes can be expanded with a double-click.
 When a field is selected, its value can be edited at the bottom of the Scene Tree.
-Double-clicking or pressing the `Enter` key on a field selects the first editable item of the field editor panel.
+Double-clicking or pressing the <kbd>enter</kbd> key on a field selects the first editable item of the field editor panel.
 Keyboard focus can be returned to the Scene Tree by tabbing through all of the items in the field editor panel.
-For text fields, changes are applied by pressing the `Enter` key.
+For text fields, changes are applied by pressing the <kbd>Enter</kbd> key.
```
Comment (stefaniapedrazzi) [inline docs/guide/the-scene-tree.md:27]:
```suggestion
For text fields, changes are applied by pressing the <kbd>enter</kbd> key.
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Changes looks fine.
But there are some missing closing tags and displaying issues.

**Comment by omichel** [comment]
Thank you for the review. I believe I fixed the problems you reported.

File: docs/reference/keyboard.md [hunk 2]
```diff
@@ -161,12 +161,12 @@ The function can be called up to 7 times to detect up to 7 simultaneous keys pre
 The `wb_keyboard_disable` function should be used to stop the keyboard readings.
 
 > **Note** [C++]: The keyboard predefined values are located into a (static) enumeration of the Keyboard class.
-For example, `Keyboard.CONTROL` corresponds to the *Control* key stroke.
+For example, `Keyboard.CONTROL` corresponds to the <kbd>Ctrl</kbd> key stroke.
```
Comment (stefaniapedrazzi) [inline docs/reference/keyboard.md:164]: For consistency this should be written with lower case
```suggestion
For example, `Keyboard.CONTROL` corresponds to the <kbd>ctrl</kbd> key stroke.
```

File: docs/guide/tutorial-3-appearance.md [hunk 3]
```diff
@@ -87,8 +87,8 @@ Webots offers several rendering modes available in the `View` menu.
 Then restore the plain rendering mode: `View / Plain Rendering`.
 
 Others rendering features can be helpful:
-- View Coordinates systems: `View / Optional Rendering / Show Coordinates System (Ctrl + F1)`.
-- View distance sensor rays: `View / Optional Rendering / Show DistanceSensor Rays (Ctrl + F10)`.
+- View Coordinates systems: `View / Optional Rendering / Show Coordinates System` <kbd>ctrl</kbd>-<kbd>F1</kbd>
+- View distance sensor rays: `View / Optional Rendering / Show DistanceSensor Rays` <kbd>ctrl</kbd>-<kbd>F10</kbd>)
```
Comment (stefaniapedrazzi) [inline docs/guide/tutorial-3-appearance.md:91]:
```suggestion
- View distance sensor rays: `View / Optional Rendering / Show DistanceSensor Rays` <kbd>ctrl</kbd>-<kbd>F10</kbd>
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you.
There are just two others minor fixes (one that I missed in the first review).

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**